Testscripts/Linux/Test_LIS_disk_check.sh

```diff
@@ -98,71 +98,81 @@ Install_Uninstall_LIS() {
-	else
-		LogMsg "boot and modules do not share the same partition."
-		boot_directory_target_size=$(expr $boot_directory_size - $boot_safe_data_size)
-	fi
-
-	LogMsg "modules_directory_size: $modules_directory_size"
-	LogMsg "modules_directory_target_size: $modules_directory_target_size"
-	LogMsg "boot_directory_size : $boot_directory_size"
-	LogMsg "boot_directory_target_size: $boot_directory_target_size"
-	if [ $single_partition_file_size -gt 0 ]; then
-		CreateFile "${single_partition_file_size}M" /lib/modules/file.out
-		singe_partition_created_file_size=$(ls -l --block-size=M /lib/modules/file.out | awk '{print $5}'  | tail -1)
+	if [[ $boot_partition == $root_partition ]]; then
+		LogMsg "boot and root share the same partition. We will create only one file"
+		single_partition_file_size=$(expr $root_part_avail_space - $root_part_required_space)
+		echo "single_partition_file_size $single_partition_file_size"
+		CreateFile "${single_partition_file_size}" /lib/modules/file.out
+		singe_partition_created_file_size=$(ls -l /lib/modules/file.out | awk '{print $5}'  | tail -1)
+		echo "singe_partition_created_file_size $singe_partition_created_file_size"
 		LogMsg "singe_partition_created_file_size : $singe_partition_created_file_size"
-		if [[ "$singe_partition_created_file_size" == "${single_partition_file_size}M" ]]; then
+		if [[ "$singe_partition_created_file_size" == "${single_partition_file_size}" ]]; then
 			file_status="created"
 		else
 			file_status="failed"
 		fi
 	else
-		CreateFile "${modules_directory_target_size}M" /lib/modules/file.out
-		module_created_file_size=$(ls -l --block-size=M /lib/modules/file.out | awk '{print $5}'  | tail -1)
-		CreateFile "${boot_directory_target_size}M" /boot/file.out
-		boot_created_file_size=$(ls -l --block-size=M /boot/file.out | awk '{print $5}'  | tail -1)
-		if [[ "$module_created_file_size" == "${modules_directory_target_size}M"
-			&& "$boot_created_file_size" == "${boot_directory_target_size}M" ]]; then
+		LogMsg "boot and modules do not share the same partition."
+		single_partition_file_size=$(expr $root_part_avail_space - $root_part_required_space)
+		single_partition_file_size_boot=$(expr $boot_part_avail_space - $boot_part_required_space)
+		echo "file size - root partition: $single_partition_file_size boot partition: $single_partition_file_size_boot"
+		CreateFile "${single_partition_file_size}" /lib/modules/file.out
+		singe_partition_created_file_size=$(ls -l /lib/modules/file.out | awk '{print $5}'  | tail -1)
+		CreateFile "${single_partition_file_size_boot}" /boot/file.out
+		boot_created_file_size=$(ls -l /boot/file.out | awk '{print $5}'  | tail -1)
+		if [[ "$single_partition_file_size" == "${singe_partition_created_file_size}"
+			&& "$single_partition_file_size_boot" == "${boot_created_file_size}" ]]; then
 			file_status="created"
 		else
 			file_status="failed"
 		fi
 	fi
+
+	stat -f  /
```

Comment:
what is the role of this command?
output is not saved anywhere, and it doesn't look to do any relevant operation?